### PR TITLE
Docker Desktop 3.2.2 release notes

### PR DESCRIPTION
**Pull request by StefanScherer**
Signed-off-by: Stefan Scherer <stefan.scherer@docker.com>

Release notes for Docker Desktop 3.2.2.

### CODE REVIEW

File: docker-for-mac/release-notes.md [hunk 1]
```diff
@@ -16,10 +16,25 @@ This page contains information about the new features, improvements, known issue
...
+
+### Bug fixes and minor changes
+
+- Allow exposing port 53 (DNS) without clashing with an internal DNS server. Fixes [docker/for-mac#5416](https://github.com/docker/for-mac/issues/5416).
+- Fixed an issue related to running iTerm that has no window open [docker/roadmap#98](https://github.com/docker/roadmap/issues/98#issuecomment-791927788)
+- Fixed an issue related to high CPU consumption and frozen UI when network connection lost [for-win/#10583](https://github.com/docker/for-win/issues/10563)
+
+### Known issue
+
+The new Apple Virtualization framework uses port 53 (DNS) when Docker Desktop is started. This port cannot be used to bind a container port to the host.
```
Comment (stephen-turner) [inline docker-for-mac/release-notes.md:32]: We don't need to document this as the virtualization framework is experimental only. (We should put it on the M1 page, but not here).

File: docker-for-mac/release-notes.md [hunk 2]
```diff
@@ -16,10 +16,26 @@ This page contains information about the new features, improvements, known issue
...
 {: .important }
 
+## Docker Desktop 3.2.2
+2021-03-15
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+### Bug fixes and minor changes
+
+- Allow exposing port 53 (DNS) without clashing with an internal DNS server. Fixes [docker/for-mac#5416](https://github.com/docker/for-mac/issues/5416) [docker/for-mac#5453](https://github.com/docker/for-mac/issues/5453).
```
Comment (stephen-turner) [inline docker-for-mac/release-notes.md:26]: "Fixed an issue that stopped containers binding to port 53".

Also we don't need to cite every duplicate ticket, just include the main one (5416).

File: docker-for-mac/release-notes.md (hunk 1 above)
Comment (StefanScherer) [reply docker-for-mac/release-notes.md:33]: Alright, I'll remove it here and create a separate PR.

File: docker-for-mac/release-notes.md [hunk 3]
```diff
@@ -16,10 +16,22 @@ This page contains information about the new features, improvements, known issue
...
+- Fixed an issue related to running iTerm that has no window open. Fixes [docker/roadmap#98](https://github.com/docker/roadmap/issues/98#issuecomment-791927788).
+- Fixed an issue running 32bit Intel binaries in emulation on Intel CPU's. Fixes [docker/for-win#10594](https://github.com/docker/for-win/issues/10594) [docker/for-win#10604](https://github.com/docker/for-win/issues/10604) [docker/for-win#10632](https://github.com/docker/for-win/issues/10632).
```
Comment (stephen-turner) [inline docker-for-mac/release-notes.md:28]: "Fixed an issue that caused 32-bit Intel binaries to be run in emulation on Intel CPUs".

And again, just choose whichever ticket is the main one, not whichever have been closed as dupes.

**Review by stephen-turner — CHANGES_REQUESTED** [review]

File: docker-for-mac/release-notes.md (hunk 1 above)
Comment (StefanScherer) [reply docker-for-mac/release-notes.md]: Updated and created #12499

File: docker-for-mac/release-notes.md [hunk 4]
```diff
@@ -16,10 +16,22 @@ This page contains information about the new features, improvements, known issue
...
+## Docker Desktop 3.2.2
+2021-03-15
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+### Bug fixes and minor changes
+
+- Allow exposing port 53 (DNS) without clashing with an internal DNS server. Fixes [docker/for-mac#5416](https://github.com/docker/for-mac/issues/5416).
+- Fixed an issue related to running iTerm that has no window open. Fixes [docker/roadmap#98](https://github.com/docker/roadmap/issues/98#issuecomment-791927788).
+- Fixed an issue running 32bit Intel binaries in emulation on Intel CPU's. Fixes [docker/for-win#10594](https://github.com/docker/for-win/issues/10594).
```
Comment (usha-mandya) [inline docker-for-mac/release-notes.md:28]:
```suggestion
- Fixed an issue running 32-bit Intel binaries in emulation on Intel CPUs. Fixes [docker/for-win#10594](https://github.com/docker/for-win/issues/10594).
```

File: docker-for-windows/release-notes.md [hunk 5]
```diff
@@ -16,10 +16,21 @@ This page contains information about the new features, improvements, known issue
...
 
+## Docker Desktop 3.2.2
+2021-03-15
+
+> [Download](https://desktop.docker.com/win/stable/amd64/Docker%20Desktop%20Installer.exe)
+
+### Bug fixes and minor changes
+
+- Allow exposing port 53 (DNS) without clashing with an internal DNS server. Fixes [docker/for-win#10601](https://github.com/docker/for-win/issues/10601).
+- Fixed an issue running 32bit Intel binaries in emulation on Intel CPU's. Fixes [docker/for-win#10594](https://github.com/docker/for-win/issues/10594).
```
Comment (usha-mandya) [inline docker-for-windows/release-notes.md:27]: Same comment as above

File: docker-for-windows/release-notes.md [hunk 6]
```diff
@@ -16,10 +16,21 @@ This page contains information about the new features, improvements, known issue
...
+- Fixed an issue related to high CPU consumption and frozen UI when network connection lost. Fixes [for-win/#10583](https://github.com/docker/for-win/issues/10563).
```
Comment (usha-mandya) [inline docker-for-windows/release-notes.md:28]:
```suggestion
- Fixed an issue related to high CPU consumption and frozen UI when network connection is lost. Fixes [for-win/#10583](https://github.com/docker/for-win/issues/10563).
```

Comment (usha-mandya) [reply docker-for-windows/release-notes.md:28]: Is there a typo in the GitHub issue number? Should it be 10563?

File: docker-for-mac/release-notes.md (hunk 4 above)
Comment (StefanScherer) [reply docker-for-mac/release-notes.md:28]: Thanks. I'll update that.

File: docker-for-windows/release-notes.md (hunk 6 above)
Comment (StefanScherer) [reply docker-for-windows/release-notes.md:28]: 👀 Oh yes, that was a typo. Thank you!

Comment (stephen-turner) [reply docker-for-windows/release-notes.md]: Yes, 63 (as in the link) not 83 (as in the text).

**Review by usha-mandya — APPROVED** [review]
LGTM

**Comment by stephen-turner** [comment]
@StefanScherer I'm reviewing it again, but you haven't taken my previous suggestions even though you gave them a 👍. Did you disagree with them, or did you just not notice them?

**Comment by StefanScherer** [comment]
@stephen-turner Thanks for double checking. Oh I missed that one, I was concentrated on removing the duplicates. 😅 
I also moved the iTerm fix at the end of the list.

File: docker-for-mac/release-notes.md [hunk 7]
```diff
@@ -16,10 +16,22 @@ This page contains information about the new features, improvements, known issue
...
 
+## Docker Desktop 3.2.2
+2021-03-15
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+### Bug fixes and minor changes
+
+- Fixed an issue that stopped containers binding to port 53. Fixes [docker/for-mac#5416](https://github.com/docker/for-mac/issues/5416).
+- Fixed an issue running 32-bit Intel binaries in emulation on Intel CPUs. Fixes [docker/for-win#10594](https://github.com/docker/for-win/issues/10594).
```
Comment (stephen-turner) [inline docker-for-mac/release-notes.md:27]: This makes it sound like they should run in emulation but it was broken. Change to:
"Fixed an issue that 32-bit Intel binaries were emulated on Intel CPUs"

File: docker-for-mac/release-notes.md [hunk 8]
```diff
@@ -16,10 +16,22 @@ This page contains information about the new features, improvements, known issue
...
+- Fixed an issue related to high CPU consumption and frozen UI when network connection is lost. Fixes [for-win/#10563](https://github.com/docker/for-win/issues/10563).
```
Comment (stephen-turner) [inline docker-for-mac/release-notes.md:28]: when the network connection

File: docker-for-mac/release-notes.md [hunk 9]
```diff
@@ -16,10 +16,22 @@ This page contains information about the new features, improvements, known issue
...
+- Fixed an issue related to running iTerm that has no window open. Fixes [docker/roadmap#98](https://github.com/docker/roadmap/issues/98#issuecomment-791927788).
```
Comment (stephen-turner) [inline docker-for-mac/release-notes.md:29]: Fixed an issue opening a terminal in iTerm2 when it has no other windows open.

**Review by stephen-turner — CHANGES_REQUESTED** [review]

File: docker-for-mac/release-notes.md (hunk 9 above)
Comment (StefanScherer) [reply docker-for-mac/release-notes.md]: Updated.

File: docker-for-mac/release-notes.md (hunk 8 above)
Comment (StefanScherer) [reply docker-for-mac/release-notes.md]: Updated in both files.

File: docker-for-mac/release-notes.md (hunk 7 above)
Comment (StefanScherer) [reply docker-for-mac/release-notes.md]: Updated in both files.

**Review by stephen-turner — APPROVED** [review]
Thanks for the changes, looks good now.

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**